proto/osmosis/gamm/pool-models/balancer/tx.proto

```diff
@@ -6,7 +6,7 @@ import "osmosis/gamm/pool-models/balancer/balancerPool.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v7/x/gamm/pool-models/balancer";
 
-service Msg {
+service BalancerMsg {
```

Comment:
Can you explain how/where this service is used? Does this actually process on-chain messages?

Reply:
It should process on-chain messages at least, I have not confirmed on a testnet. The service name as far as I can tell is just for generated protobuf structs, we have methods to register the proto service, linking the go struct to the proto message

Reply:
Sorry if this is obvious, but why are they not all under a single `service?

Reply:
https://github.com/bufbuild/buf/issues/99#issuecomment-655841579